frontends/common/setup.h

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
Yeah, maybe that would be simplest; it does seem to belong with `CompilerOptions::preprocess()`.